runtime/tutorials/subprocess.md

```diff
@@ -49,23 +49,6 @@ The `--allow-run` permission is required for creation of a subprocess. Be aware
 that subprocesses are not run in a Deno sandbox and therefore have the same
 permissions as if you were to run the command from the command line yourself.
 
-Be very careful using this permission as in many cases it does not provide much
-security. For example:
-
-1. Using `--allow-run` without an allow list is essentially the same as
-   `--allow-all`. A script could execute the `deno` executable with full
-   permissions (ex. `deno eval '<malicious code goes here>'`).
-1. Using `--allow-run=... --allow-write` means a script could overwrite an
-   executable on the path and execute it.
-1. Using `--allow-run=... --allow-env=PATH` and any `--allow-write=...` means a
-   script could modify the `PATH` to a writable location, then write an
-   executable and execute it.
-
-Even when locking down some of the scenarios listed above there are still
-possible exploits (ex. when combining options with `--deny-*` flags). Think
-through the combination of flags that you use carefully when using
-`--allow-run`!
```

Comment:
I updated this recently in https://github.com/denoland/docs/pull/750 and I think the information in fundamentals/security.md is sufficient.